packages/asc-ui/src/components/Button/ButtonStyle.ts

```diff
@@ -80,6 +80,7 @@ const getVariant = () => ({
     case 'blank':
       return css`
         ${svgFill('tint', 'level7')}
+        background-color: ${getColor('tint', 'level1')({ theme })};
```

Comment:
Does the `getColor` function has to curry the `theme`? As `svgFill` doesn't seem to do so